src/core-interfaces.ts

```diff
@@ -23,6 +23,10 @@ export type ShapeFill = Color | { type: string; color: Color; alpha?: number }
 export type BkgdOpts = { src?: string; path?: string; data?: string }
```

Comment:
@gitbrent would you mind explaining your thinking behind having both `core-*.ts` files as well as `types/index.d.ts`? My assumption is that a goal would be to get rid of one of these files and have types coming from one place?

Reply:
I dug in a bit more and noticed that `tsconfig.js` has `declarationDir` set (https://github.com/gitbrent/PptxGenJS/blob/master/tsconfig.json#L4). 

I think that is supposed to output a types definition file when ts compiles which would mean the `types/index.d.ts` would not be needed. Will look into why that file is not being created in the `out` directory any longer.

Reply:
@gitbrent I haven't had a chance yet to track down the `delcarationDir` issues. I think I could look into generating the types once in another PR.